skopt/gp_opt.py

```diff
@@ -246,6 +296,9 @@ def gp_minimize(func, dimensions, base_estimator=None, alpha=10e-10,
     models = []
     n_model_iter = n_calls - n_total_init_calls
     for i in range(n_model_iter):
+
```

Comment:
Do you have a github emoji guide or is there a dropdown that I am missing somewhere?